application/src/main/java/org/togetherjava/tjbot/commands/generic/PingCommand.java

```diff
@@ -0,0 +1,30 @@
+package org.togetherjava.tjbot.commands.generic;
+
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.togetherjava.tjbot.commands.example.AbstractCommand;
+
+/**
+ * Ping Pong command.
+ */
+public class PingCommand extends AbstractCommand {
+    private static final Logger logger = LoggerFactory.getLogger(PingCommand.class);
+
+    public PingCommand() {
+        super("ping",
+                "This command returns a message with the value `pong`, to show the user that the bot is up and running.");
```

Comment:
Descriptions have a max length of 100 chars

Reply:
could we cover this in the code with an exception and javadoc?